cmd/sf/sf.go

```diff
@@ -427,7 +427,7 @@ func main() {
 	case *jsono:
 		w = writer.JSON(os.Stdout)
 	case *droido:
-		if len(s.Fields()) != 1 || len(s.Fields()[0]) != 7 {
+		if len(s.Fields()) != 1 || len(s.Fields()[0]) != 8 {
```

Comment:
I was thinking about something like this for here: 

```golang
	p, _ := pronom.New()
	if len(s.Fields()) != 1 || len(s.Fields()[0]) != len(p.Fields()) {
	    // ...
	}
```
But is that just creating a big overhead in terms of speed (I think all reports are read?) and there's always the potential for error. Is there another way to export the fields as they're constant?

Reply:
yes, loading pronom here would be expensive!